tests/integration/_experimental/search/test_embeddings.py

```diff
@@ -62,10 +69,79 @@ def test_upload_embeddings(embedding: np.ndarray) -> None:
         np.array([1, "2"]),
     ],
 )
-def test_upload_embeddings__bad_embedding(embedding: np.ndarray) -> None:
+def test__upload_embeddings__bad_embedding(embedding: np.ndarray) -> None:
     locator = fake_random_locator()
     with pytest.raises(InputValidationError):
         upload_embeddings(
             key="s3://model-bucket/embeddings-model.pt",
             embeddings=[(locator, embedding)],
         )
+
+
+@pytest.fixture(scope="module", autouse=True)
+def dataset_name() -> str:
+    name = with_test_prefix(f"{__file__}::test__embedding_dataset {uuid.uuid4()}")
+    datapoints = [dict(locator=f"locator-{i}", value=i) for i in range(N_DATAPOINTS)]
+    upload_dataset(name, pd.DataFrame(datapoints), id_fields=["locator"])
+    return name
+
+
+@pytest.mark.skip("disabled until server side changes are deployed")
```

Comment:
Will uncomment these tests after https://github.com/kolenaIO/shipyard/pull/1823 is in production